src/Uno.UWP/System/Launcher.iOS.SpecialUris.cs

```diff
@@ -0,0 +1,36 @@
+﻿#if __IOS__
```

Comment:
You seems to have an implementation for iOS, but the documentation is only about Android.

Reply:
I mention there that "In case of iOS, any such API opens the main page of system settings.", but I will add it to separate paragraph to make it more visible

Reply:
Done :-)